main.tf

```diff
@@ -73,6 +73,7 @@ resource "aws_ec2_transit_gateway_vpc_attachment" "this" {
 
   dns_support                                     = lookup(each.value, "dns_support", true) ? "enable" : "disable"
   ipv6_support                                    = lookup(each.value, "ipv6_support", false) ? "enable" : "disable"
+  appliance_mode_support                          = lookup(each.value, "appliance_mode_support", false) ? "enable" : "disable"
```

Comment:
This feature became in version 3.15.0 ( https://github.com/hashicorp/terraform-provider-aws/blob/main/CHANGELOG.md#3150-november-12-2020 ).

Could you please update `versions.tf` also to reflect this as a requirement - https://github.com/terraform-aws-modules/terraform-aws-transit-gateway/blob/master/versions.tf#L5 ?

Once done, you can run `pre-commit run -a` to update the docs.

Reply:
done and done 😄